packages/agent/src/hooks/useDidCommPresentationActions.ts

```diff
@@ -94,8 +111,11 @@ export function useDidCommPresentationActions(proofExchangeId: string) {
           // This should probably be fixed in AFJ.
           const firstMatch = predicateArray[0]
 
+          // When the credentialId isn't available, we use the groupName as the key but it will result in multiple entries in the view. But I think it's not an easy task to merge them
+          const credentialKey = firstMatch?.credentialId ?? groupName
```

Comment:
Maybe if no credentialId is available we match based on the CREDENTIAL_ prefix like you had before as fallback?

Otherwise there's not an easy way to group them

Reply:
Sure brought it back 😄